packages/api-bindings/src/lens/common.graphql

```diff
@@ -470,6 +471,7 @@ fragment Comment on Comment {
 fragment Post on Post {
   __typename
   id
+  appId
```

Comment:
could you add it also in the client sdk?

Reply:
Sure can